.github/workflows/tests.yml

```diff
@@ -43,7 +48,7 @@ jobs:
             PYTHON_VERSION: '3.11'
             PIP_SELECTOR: '[all, tests, coverage]'
           - os: macos
-            os_version: '14'
+            os_version: '13'
```

Comment:
Why run on a lower macos version?

Reply:
To test on both architectures currently available on mac: intel and arm.